static/js/src/advantage/subscribe/checkout/components/UserInfoForm/UserInfoForm.tsx

```diff
@@ -396,6 +396,10 @@ const UserInfoForm = ({ setError }: Props) => {
             validate={validateRequired}
             error={touched?.postalCode && errors?.postalCode}
           />
+          <p>
+            We will save your card information. You can change it in your Ubuntu
+            Pro Dashboard.
+          </p>
```

Comment:
Distributor users don't have access to the Pro dashboard, but this message is also showing in the checkout for them. Should we display this differently for distributor users?